tutorials/shaders/shader_reference/shader_functions.rst

```diff
@@ -770,18 +769,21 @@ Exponential Function Descriptions
 
 .. rst-class:: classref-method
 
-| |vec_type| **abs**\ (\ |vec_type| x)
-| |vec_int_type| **abs**\ (\ |vec_int_type| x)
+|vec_type| **abs**\ (\ |vec_type| x)
+
+.. rst-class:: classref-method
```

Comment:
This notation is considerably more verbose than my original using the bars and I can't see any difference in the rendered HTML. Any particular reason why we need to change it?

Reply:
That's weird! When I built it locally, I noticed two problems with how it rendered the overloaded function entries:
- The function names did not appear as white, but as gray.
- The wrapping was incorrect, so the return type of the function appeared at the end of the line.

I didn't get any screenshots, though. I'm going to rebuild and get some screenshots, then post a comment to the original PR so we don't fragment the review.

Reply:
Ok, tomorrow I can take some screenshots from my computer too if that would help

Reply:
Yeah, if both ways render identically for you, this might be something platform dependent. You should test on `clamp`, `textureGrad`, or `textureProjGrad`. For context, for me these look like this with this PR applied, on a 1080 screen, Firefox, Windows 10, 100% zoom in browser:
![firefox_Ds37GehAo5](https://github.com/user-attachments/assets/9069e910-363e-413d-913d-119c377df9a9)
![firefox_lcQQPZ0Iu0](https://github.com/user-attachments/assets/722928bf-3be3-469b-ace7-7ebeb9e83966)
![firefox_twHKSnvJ6o](https://github.com/user-attachments/assets/0a915f50-c85a-455f-b284-c4efd017137a)